dashboard-for-desktop/articles/dashboard-designer/designing-dashboard-items/chart.md

```diff
@@ -32,4 +32,7 @@ This section is divided into the following subsections.
 	Provides information about the chart legend and its options.
 * [Orientation](chart/orientation.md)
 	
```

Comment:
Since I cannot edit the above text, can you please revise the following lines?

Line 14: `Provides information on how to supply the Chart dashboard item with data.`
I'd like to update this without using "Provides". Can we say:

`Explains how to supply the Chart dashboard item with data.`

Line 20: `Introduces the concept of chart panes (visual areas within a diagram that display chart series), and provides information on how to create them.`
Change to:

`Introduces the concept of chart panes (visual areas within a diagram that display chart series), and describes how to create them.`